CONTRIBUTING.md

```diff
@@ -0,0 +1,157 @@
+[![Build Status](https://travis-ci.com/PhoenicisOrg/scripts.svg?branch=master)](https://travis-ci.com/PhoenicisOrg/scripts)
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/ff0c41daa31549e4a9bb3998ca0c87ae)](https://www.codacy.com/app/PhoenicisOrg/scripts?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=PhoenicisOrg/scripts&amp;utm_campaign=Badge_Grade)
+[![Crowdin Badge](https://d322cqt584bo4o.cloudfront.net/phoenicis-scripts/localized.svg)](https://crowdin.com/project/phoenicis-scripts)
+
+# How to add a script
+1. Fork this repository, create a new branch and clone it 
+    ```
```

Comment:
I think there is `bash` syntax highlighting

Reply:
added